### PR TITLE
update readme with workflow documentation

### PR DESCRIPTION
**Pull request by carlosaicrag**
update read me docs with workflow documentation.  Also added some more description to the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,25 +4,46 @@
...
-* there will need to be at least one review before it gets pulled into the master branch
-
+1. git pull master 
+2. git checkout -b `<name of branch>`
+3. work on feature 
+4. once you are done 
+    * checkout the master branch
+    * git pull on the master branch
+    * checkout YOUR feature branch
+    * git merge master
```
Comment (linakherchi) [inline README.md]: Do we need to fix conflicts first, add, commit, and then push ?

File: README.md [hunk 2]
```diff
@@ -4,25 +4,46 @@
...
+    * git pull on the master branch
+    * checkout YOUR feature branch
+    * git merge master 
+    * git push
+    * do the command that the terminal says to do
+    * your branch should now be in the remote repository
+5. create a pull request on github 
+    * ![pull_request1]( ./images/pull_request1.png )
+    * ![pull_request2]( ./images/pull_request2.png )
+    * ![pull_request3]( ./images/pull_request3.png )
```
Comment (linakherchi) [inline README.md:45]: I am not able to see these images

File: README.md (hunk 1 above)
Comment (carlosaicrag) [reply README.md:38]: you're right! just included that

File: README.md (hunk 2 above)
Comment (carlosaicrag) [reply README.md:45]: Thats weird cuz i can see it on my end

**Pull request merged**